<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectEfitNe</class>
 <widget class="QWidget" name="SelectEfitNe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>516</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Efit From:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbFile">
        <property name="text">
         <string>File</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbMdsPlus">
        <property name="text">
         <string>MDS+ Server</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gfile Dir:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leGFileDir">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tbGFileDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leDataDir">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Data Dir:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="tbData">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>efit_east</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>efitrt_east</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame_3">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QCheckBox" name="diagnostics2">
        <property name="text">
         <string>Thomson scattering(TS)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="diagnostics1">
        <property name="text">
         <string>Reflectometry(ReflJ)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>257</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMdsPlus</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMdsPlus</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_4</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMdsPlus</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_4</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
